app/src/main/java/org/fossasia/openevent/general/auth/AuthApi.kt

```diff
@@ -14,4 +11,7 @@ interface AuthApi {
     @GET("users/{id}")
     fun getProfile(@Path("id") id: Long): Single<User>
 
+    @POST("users")
+    fun signUp(@Body user: User, @Header("Content-Type")header : String): Single<User>
```

Comment:
is it needed sending content-type?

Reply:
Yeah okhttp sends an error of missing content type header

Reply:
but there would be another way to do it ? lets wait for the mentor to weigh in

Reply:
Yeah sure

Reply:
No it's not needed. See the orga app

Reply:
Okay :+1: